files/environments/jupyterlab.yaml

```diff
@@ -13,9 +13,17 @@ dependencies:
   # dask
   - dask-gateway
   - dask-labextension>=5.0.0
-  # dashboard
+  # panel
+  - jupyter-panel-proxy
```

Comment:
Could you remove jupyter-panel-proxy? Tried to remove this in favor of your work.